websites/T/Tempest Fansub/presence.ts

```diff
@@ -0,0 +1,129 @@
+const presence = new Presence({
+		clientId: "1047102386478534727",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+presence.on("UpdateData", async () => {
+	const { pathname } = document.location,
+		presenceData: PresenceData = {
+			startTimestamp: browsingTimestamp,
+			largeImageKey: "https://i.imgur.com/MVWiX8p.png",
+		},
+		searchParams = new URLSearchParams(location.search);
+	//Dino
+	if (pathname === "/dinozor-oyunu/") presenceData.details = "Dino oynuyor.";
+	//arama
+	else if (searchParams.get("s")) {
+		if (location.host === "tempestfansub.com")
```

Comment:
Probably best to use `hostname (document.location.hostname)` and add it to the constant above with `pathname`

Reply:
Thanks for the explanatory response.